src/foam/nanos/auth/email/ClientEmailVerificationService.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * @license
+ * Copyright 2022 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
```

Comment:
This isn't needed anymore. Just add:
  client: true,

in EmailVerificationService.